src/webln/NostrWeblnProvider.ts

```diff
@@ -277,10 +278,26 @@ export class NostrWebLNProvider implements WebLNProvider {
     if (options?.returnTo) {
       url.searchParams.set('return_to', options.returnTo);
     }
+
+    if (options.budgetRenewal) {
+      url.searchParams.set("budget_renewal", options.budgetRenewal)
+    }
+    if (options.expiresAt) {
+      url.searchParams.set("expires_at", options.expiresAt.toISOString())
```

Comment:
what do you think about using a timestamp instead of the string repesentation?

Reply:
@bumi I think it would be good to support both. Would that be milliseconds (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/now) or seconds since the epoch?

nostr-wallet-connect only supports a string representation right now. I will create an issue there. I think this could be improved in a follow up PR?

Reply:
both is just adding complexity but not really making it much better. It is easy to make a timestamp in any language and a timestamp also does not have the confusion of potentially different string representations with wrong timezones for example. 

seconds is the default typically afaik - and good enough.

Reply:
@bumi updated